flake.nix

```
@@ -55,6 +55,8 @@
       pcengines-apu = import ./pcengines/apu;
       raspberry-pi-2 = import ./raspberry-pi/2;
       samsung-np900x3c = import ./samsung/np900x3c;
+      system76-darp6 = import ./system76/darp6;
+      system76-generic = import ./system76/generic;
```

We have this convention in nixos-hardware:

```suggestion
      system76 = import ./system76;
      system76-darp6 = import ./system76/darp6;
```

---

I see.  I'm going to simplify this down to just one `system76/default.nix` module, with no options and just including all of the modules.  From what I can tell it doesn't cause problems to have all three modules enabled, and the drivers themselves are small (~160KiB total).